packages/api/src/services/security/security.service.ts

```diff
@@ -50,4 +53,48 @@ export class SecurityService implements Service {
     return this.securityRestService.getAuthenticatedUser()
       .then((response) => MapperProvider.get(AuthenticatedUserMapper).mapToModel(response));
   }
+
+  /**
+   * Checks if password-based login is enabled in the current security configuration.
+   *
+   * @returns `true` if password login is enabled; `undefined` if no config is present.
+   */
+  isPasswordLoginEnabled(): boolean | undefined {
```

Comment:
Do we need to return `undefined`? Can the password login be enabled if we do not have a security config?

Same or the OpenId down.

Reply:
fixed